volumes/miovision/getting_started.md

```diff
@@ -98,3 +99,47 @@ The `leg` represents the side of the intersection that the pedestrian is crossin
 
 **In the ATR table (aka `volumes_15min`)** 
 The `leg` represents the side of the intersection that the pedestrian is crossing. The `dir` represents which direction they are walking towards. So, if leg = N and dir = EB means that the pedestrian is at the North crosswalk crossing from the west side to the east side.
+
+### Calculating Volumes on a Segment
+
+To calculate volumes along a segment, use the approach volumes. When exit volumes are used, the vehicles turning into the intersection must be taken into account, and that adds another level of complexity. 
+
+To calculate approach volumes, ensure that the leg and the direction are opposites. A vehicle travelling north (`dir = 'NB'`) approaches an intersection from the south (`leg = 'S'`). Conversely, if a vehicle is travelling north (`dir = 'NB'`) on the north leg of an intersection (`leg = 'N'`) the vehicle has exited the intersection - the leg and the direction are the same!
+
+#### See it in Code!
+To calculate all vehicle volumes on an East-West street with traffic in both directions, add the west bound traffic on the east leg to the east bound traffic on the west leg. The code snippet below calculates the average, minimum and maximum weekday vehicle volumes for King and Bathurst (`intersection_uid = 10`) and King and Spadina (`intersection_uid = 12`) in October 2023.
+```
+WITH daily_volumes AS (
+    SELECT
+        i.intersection_uid,
+        i.intersection_name,
+        date_trunc('day', datetime_bin) AS dt,
+        SUM(volume) AS daily_vol
+    FROM miovision_api.intersections AS i
+    JOIN miovision_api.volumes_15min AS dv
+        USING (intersection_uid)
+    WHERE
+        i.intersection_uid IN (10, 12)
+        AND classification_uid = 1 --vehicles
+        AND dv.datetime_bin >= '2023-10-01'::date
+        AND dv.datetime_bin < '2023-11-01'::date
+        AND date_part('isodow', datetime_bin) <= 5 -- weekdays
+        AND ((dv.leg = 'E' AND dv.dir = 'WB')
+            OR (dv.leg = 'W' AND dv.dir = 'EB'))
+    GROUP BY
+        i.intersection_uid,
+        i.intersection_name,
+        dt
+	HAVING COUNT(DISTINCT datetime_bin) > 92
```

Comment:
volume will be `NULL` for time bins that have been processed but where we determined the data to be dubious/gaps.